src/main/java/gov/nasa/cumulus/metadata/aggregator/MetadataAggregatorLambda.java

```diff
@@ -226,15 +228,16 @@ private String postIngestProcess(String input)
 		// Set CMR related metadata into ECHOResetClientProvider
 		CMRLambdaRestClient elrc = buildLambdaRestClient(input);
 		setCMRMetadataToProvider(input);
-		// From this point, determine if we are going to process Footprint (fp) only
-		if(FootprintProcessor.isFootprintFileExisting(input)) {
+		// From this point, determine if we are going to process Forge workflow
+		if(workflowType == WorkflowTypeEnum.ForgeWorkflow) {
```

Comment:
where is workflowType set?

Reply:
Totally missed that part; added it back in (line 51)